service-front/module/Application/src/Controller/CPFlowController.php

```diff
@@ -618,4 +621,41 @@ public function chooseCountryAction(): ViewModel|Response
 
         return $view->setTemplate($templates['default']);
     }
+
+    public function chooseCountryIdAction(): ViewModel|Response
+    {
+        $templates = ['default' => 'application/pages/cp/choose_country_id'];
+        $uuid = $this->params()->fromRoute("uuid");
+        $view = new ViewModel();
+        $detailsData = $this->opgApiService->getDetailsData($uuid);
+        $idOptionsData = $this->config['opg_settings']['non_uk_identity_methods'];
+        $idCountriesData = $this->config['opg_settings']['acceptable_nations_for_id_documents'];
+        $docs = $this->localisationHelper->getInternationalSupportedDocuments($detailsData);
+
+        $form = (new AttributeBuilder())->createForm(CountryDocument::class);
+        $view->setVariable('form', $form);
+
+        if ($this->getRequest()->isPost()) {
+            $form->setData($this->getRequest()->getPost());
+            $formData = $this->getRequest()->getPost()->toArray();
+
+            if ($form->isValid()) {
+                $responseData = $this->opgApiService->updateIdMethodWithCountry($uuid, $formData);
+                if ($responseData['result'] === 'Updated') {
```

Comment:
We should just be checking that the response code is 2xx, rather than passing around random case-sensitive strings